src/main/kotlin/no/nav/familie/ks/sak/barnehagelister/domene/BarnehagelisteMottatt.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.familie.ks.sak.barnehagelister.domene
+
+import java.time.LocalDateTime
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity(name = "BarnehagelisteMottatt")
+@Table(name = "BARNEHAGELISTE_MOTTATT")
+data class BarnehagelisteMottatt(
+    @Id
+    val id: UUID = UUID.randomUUID(),
+
+    @Column(name = "MELDING_ID", nullable = false, updatable = false, unique = true)
+    val meldingId: String,
+
+    @Column(name = "MELDING", nullable = true)
+    var melding: String,
+
+    @Column(name = "MOTTATT_TID", nullable = false)
+    var mottat_tid: LocalDateTime,
```

Comment:
Siden vi har brukt camelCase fram til nå, kanskje fortsette med det her?
```suggestion
    var mottattTid: LocalDateTime,
```